Source/Scene/Model.js

```diff
@@ -1952,53 +1963,20 @@ function parseTextures(model, context, supportsWebP) {
       supportsWebP
     ) {
       imageId = texture.extensions.EXT_texture_webp.source;
+    } else if (
+      defined(texture.extensions) &&
+      defined(texture.extensions.KHR_texture_basisu) &&
+      FeatureDetection.supportsBasis
```

Comment:
Related to the other comment, this should use `context.supportsBasis` since `Model` doesn't always have access to `scene`.